json_serializable/tool/build_actions.dart

```diff
@@ -5,50 +5,46 @@
 import 'dart:async';
```

Comment:
@natebosch – want to update to the new `run` method and go from 3 -> 1 file? Or in a follow-up?

Reply:
Done